3.10/aql/invocation-with-arangosh.md

```diff
@@ -242,6 +242,44 @@ In the ArangoDB Enterprise Edition there is an additional parameter:
   into sync. The default value is 60.0 (seconds). When the max time has been reached
   the query will be stopped.
 
+Additional parameters for spilling data from the query onto disk
```

Comment:
None of the other options have their own headline. I wonder whether we should group some of the others as well, or not do that for any of them. Giving each option an own headline would be another option and allow to link to each single one (we can hide them in the ToC if necessary)

Reply:
Actually, the options above this are explained one by one starting in line 141, so I added these extra ones too